src/main/webapp/resources/js/pages/analysis/components/AnalysisDetails.jsx

```diff
@@ -0,0 +1,197 @@
+        title: getI18N("analysis.tab.content.analysis.id"),
+        desc: analysisContext.analysis.identifier,
+      },
+      {
+        title: getI18N("analysis.tab.content.analysis.pipeline"),
+        desc: `${analysisDetailsContext.workflowName} (${analysisDetailsContext.version})`
+      },
+      {
+        title: getI18N("analysis.tab.content.analysis.priority"),
+        desc: analysisDetailsContext.priority
+      },
+      {
+        title: getI18N("analysis.tab.content.analysis.created"),
+        desc: formatDate({ date: analysisDetailsContext.createdDate })
+      },
+      {
+        title: getI18N("analysis.tab.content.analysis.duration"),
+        desc: getHumanizedDuration({ date: analysisDetailsContext.duration })
+      },
+    ];
+
+    /*
+        On change of checkbox to receive/not receive an email upon
+        pipeline completion update emailPipelineResult field
+    */
+    function updateEmailPipelineResult(e){
+        updateAnalysisEmailPipelineResult(analysisContext.analysis.identifier, e.target.checked).then(res => {
+          showNotification({ text: res.message});
+          dispatch({ type: 'UPDATED_EMAIL_PIPELINE_RESULT', emailPipelineResult: e.target.checked })
+        });
+    }
+
+    // Update analysis name
+    function updateSubmissionName(){
+        const updatedAnalysisName = document.getElementById("analysis-name").value.trim();
+
+        if((updatedAnalysisName  !== "") && (updatedAnalysisName !== analysisDetailsContext.analysisName)) {
+            updateAnalysis(analysisContext.analysis.identifier, updatedAnalysisName, null).then(res => {
+                showNotification({ text: res.message});
+                analysisContextUpdateSubmissionName(updatedAnalysisName);
+            });
+        }
+    }
+
+    // Update analysis priority
+    function updateAnalysisPriority(updatedPriority) {
+        updateAnalysis(analysisContext.analysis.identifier, null, updatedPriority).then(res => {
+            showNotification({ text: res.message});
+            dispatch({ type: 'UPDATED_PRIORITY', priority: updatedPriority });
+        });
+    }
+
+    // Render priorities in priority dropdown (Admins only)
+    function renderPriorities() {
+        const priorityList = [];
+
+        for(let priority of analysisDetailsContext.priorities) {
+            priorityList.push(
+              <Select.Option key={priority} value={priority}>{priority}</Select.Option>
+            )
+        }
+        return priorityList;
+    }
+
+  return (
+      <>
+          <Tabs defaultActiveKey="4"
+                tabPosition="left"
+                style={{marginLeft:150, paddingTop:25}}
+                animated={false}
+          >
+              <TabPane tab={getI18N("analysis.tab.content.analysis.details")} key="4" style={{minWidth:300}}>
+                  <Col span={12}>
+                    <h2 style={{fontWeight: "bold"}} className="spaced-bottom">{getI18N("analysis.tab.content.analysis.details")}</h2>
```

Comment:
Ant.design typography

Reply:
Fixed in 4d5b63fdf69f55254f7b8ec40e149f8fc26fcd4a